.tekton/acs-konflux-tasks-push.yaml

```diff
@@ -74,52 +82,87 @@ spec:
       description: Skip checks against built image
       name: skip-checks
       type: string
-    - default: "false"
-      description: Execute the build with network isolation
-      name: hermetic
-      type: string
     - default: ""
       description: Build dependencies to be prefetched by Cachi2
       name: prefetch-input
       type: string
-    - default: ""
-      description: Image tag expiration time, time values could be something like
-        1h, 2d, 3w for hours, days, and weeks, respectively.
-      name: image-expires-after
     - default: "false"
       description: Build a source image.
       name: build-source-image
       type: string
-    - default: "false"
-      description: Add built image into an OCI image index
-      name: build-image-index
-      type: string
-    - default: []
-      description: Array of --build-arg values ("arg=value" strings) for buildah
-      name: build-args
-      type: array
-    - default: ""
-      description: Path to a file with build arguments for buildah, see https://www.mankier.com/1/buildah-build#--build-arg-file
-      name: build-args-file
+    - default: "1d"
+      description: This sets the expiration time for intermediate OCI artifacts produced and used during builds after which they can be garbage collected.
+      name: oci-artifact-expires-after
       type: string
+
     results:
     - description: ""
       name: IMAGE_URL
-      value: $(tasks.build-image-index.results.IMAGE_URL)
+      value: $(tasks.build-container.results.IMAGE_URL)
     - description: ""
       name: IMAGE_DIGEST
-      value: $(tasks.build-image-index.results.IMAGE_DIGEST)
+      value: $(tasks.build-container.results.IMAGE_DIGEST)
     - description: ""
       name: CHAINS-GIT_URL
       value: $(tasks.clone-repository.results.url)
     - description: ""
       name: CHAINS-GIT_COMMIT
       value: $(tasks.clone-repository.results.commit)
+    - description: Bundle reference you should use in the pipelines
+      name: BUNDLE_REF
+      value: $(tasks.get-bundle-info.results.BUNDLE_REF)
+    - description: Bundle contents
+      name: BUNDLE_CONTENTS
+      value: $(tasks.get-bundle-info.results.BUNDLE_CONTENTS)
+
+    workspaces:
+    - name: git-auth
+    - name: netrc
```

Comment:
Do we need this workspace? We don't use it in our other pipelines.

Reply:
`netrc` came from auto-generated pipelines. Our older ones were generated without it.

As I understand, it is a mechanism to add custom authentication for `curl` in prefetch task. Since we don't use it and are unlikely to start using soon, I deleted it.